### PR TITLE
Adding an initial governance document.

### LINKED ISSUE #8: Define governance model

### PR DESCRIPTION
**Pull request by ringods**
The base of this document is not created anew, but taken from a previously succesful community: [Vox Pupuli](https://voxpupuli.org/)

Source: https://github.com/voxpupuli/plumbing/blob/master/share/governance.md

Signed-off-by: Ringo De Smet <ringo@de-smet.name>

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+* Advocating for use of the project
+* Informing developers of project strengths and weaknesses from a user’s perspective
+* Providing moral support (a ‘thank you’ goes a long way)
+* Writing documentation and tutorials
+* Filing bug reports and feature requests
+* Participating on the discussion board
+
+Users who continue to engage with the project and its community will often find
+themselves becoming more and more involved. Such users may then go on to become
+collaborators and contributors, as described below.
```
Comment (simenandre) [inline GOVERNANCE.md:46]: I don't understand this. I don't feel it's relevant for governance, does that make sense?

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+### Contributors
+
+*How to become one*: Submit a pull request to a Pulumiverse project
+
+Contributors are community members who submit patches or other contributions
+(art, docs, user support, etc) to the project. These patches may be a one-time
+occurrence or occur over time. It is anticipated that contributions may be
+small at first, growing more considerable once the contributor has built up the
+necessary knowledge, experience, and confidence.
```
Comment (simenandre) [inline GOVERNANCE.md:56]: I feel the same about this section as above.

File: GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+project in general or deep dedication to one project in particular, and the
+ability to work well with contributors and other users. The collaborators have
+responsibilities beyond the contributors. In particular, collaborators formally
+decide on whether a pull request is merged or a release is cut. Collaborators
+are recognized and valued for their contributions, in whatever form those
+contributions take. Tasks such as communication, documentation, art, packaging,
+and testing are all valued contributions. A collaborator will use lazy
+consensus to decide on whether to merge a pull request from a contributor. If
+the discussion is no longer moving towards a consensus, the PMC must vote via
+lazy consensus on whether the patch should be applied.
```
Comment (simenandre) [inline GOVERNANCE.md:78]: I feel the same about this section as above.

File: GOVERNANCE.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+Our release process for modules creates a Git tag that's signed with GPG.
+Therefor collaborators need to [setup GPG](https://help.github.com/en/articles/telling-git-about-your-signing-key)
+on their machine and add the public key to [their GitHub profile](https://help.github.com/en/articles/adding-a-new-gpg-key-to-your-github-account).
+It's also a good advice to sign [every commit](https://help.github.com/en/articles/signing-commits)
+and [tag](https://help.github.com/en/articles/signing-tags) with GPG.
```
Comment (simenandre) [inline GOVERNANCE.md:84]: I would rather have this stated in a `CONTRIBUTE.md`

File: GOVERNANCE.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+
+### Project Management Committee (PMC)
```
Comment (simenandre) [inline GOVERNANCE.md:86]: I'm not sure about the naming. Is «PMC» used broadly?

File: GOVERNANCE.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+cannot be reached.
+
+The PMC has final say over who can become a committer and will use lazy
+consensus for approval. Discussion over committer nominations will be done in
+private.
+
+Membership of the PMC is by election.
+[Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) voting is held once
+a year. A non-running election official is appointed by the PMC and runs the
+election. There are 5 members of the PMC. Terms are 1 year.
```
Comment (simenandre) [inline GOVERNANCE.md:104]: We talked about having staggering elections, and I think we should.

File: GOVERNANCE.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+A member of the PMC can be removed before the end of their 1 year term by a
+supermajority (2/3) vote of the PMC. This can happen for a number of reasons,
+but the expected reason is following a Code of Conduct violation incident. Any
+report of problematic behaviour of PMC members will trigger an investigation
+and can trigger a vote. A report of the incident and actions taken (if any)
+will be published for the public. A member of the PMC can step down at any
+time.
+
+If a vacancy is left on the PMC, the PMC can optionally instruct the elections
+officer to perform another election or wait until the next election cycle.
```
Comment (simenandre) [inline GOVERNANCE.md:168]: «Will trigger an investigation». By whom?
Who decides to trigger a vote? Anyone?

«by a supermajority (2/3) vote of the PMC». I don't think that is enough. Only PMCs are able to kick a PMC? The community needs to be able to. How?

**Review by simenandre — COMMENTED** [review]
I'm not sure this document is a great starting point. I feel it's generally off-topic.

File: GOVERNANCE.md (hunk 6 above)
Comment (simenandre) [reply GOVERNANCE.md:104]: How are elections run? Just on Slack? When are the elections?

**Comment by ringods** [comment]
After some discussion, we decided to continue with #13 as the basis for the governance documents.

**Pull request closed without merging**